src/base/gfx/pixmap.cpp

```diff
@@ -6,12 +6,12 @@ size_t PixMapView::bytesPerPixel() const { return grayscale ? 2 : 4; }
 
 size_t PixMapView::bytesPerLine() const
 {
-	return bytesPerPixel() * (int)size.x;
+	return bytesPerPixel() * static_cast<int>(size.x);
```

Comment:
PixMap and PixMapView are not used, they can be removed as well

Reply:
OK